src/pyEQL/pourbaix/pourbaix_diagram.py

```diff
@@ -964,6 +964,421 @@ def from_dict(cls, dct: dict) -> Self:
         )
 
 
+class Multielement_PourbaixDiagram(MSONable):
+    elements_ho: ClassVar[set[Element]] = {Element("H"), Element("O")}
+
+    def __init__(
+        self,
+        entries: list[PourbaixEntry] | list[MultiEntry],
+        comp_dict: list[dict[str, float]] | None = None,
```

Comment:
Let's also allow input of a single dict, just like the current behavior, e.g.,

`list[dict[str, float]] | dict | None`